Makefile

```diff
@@ -134,14 +140,14 @@ docker-push: ## Push docker image with the manager.
 # - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:<tag>> than the export will fail)
 # To properly provided solutions that supports more than one platform you should use this option.
 PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
-.PHONY: docker-buildx
-docker-buildx: test ## Build and push docker image for the manager for cross-platform support
+.PHONY: operator-buildx
+operator-buildx: test ## Build and push docker image for the manager for cross-platform support
 	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
 	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' Dockerfile > Dockerfile.cross
-	- docker buildx create --name project-v3-builder
-	docker buildx use project-v3-builder
-	- docker buildx build --push --platform=$(PLATFORMS) --tag ${IMG} -f Dockerfile.cross
-	- docker buildx rm project-v3-builder
+	- $(CTR_CMD) buildx create --name project-v3-builder
```

Comment:
Can you confirm if podman support `buildx`? Last time I checked only `buildah` had cross platform build compatibilty.
[Related issue in podman ](https://github.com/containers/buildah/issues/1590)

Reply:
I think we don't actually need this as podman and docker both supports `--platform` flag. I can update the `make operator-build` step to have a platform flag added. Wdyt?

Reply:
Yes we need to make sure that the operator bundles are created and validated correctly for podman too.